.github/workflows/leak_detect.yml

```diff
@@ -0,0 +1,40 @@
+name: Detect Project ID leak
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
```

Comment:
So since no branches are specified here, this will happen on any push or PR update?

Reply:
indeed